docs/getting-started/advanced-setting.asciidoc

```diff
@@ -64,7 +56,7 @@ data shipped via {beats} and the {agent} is automatically added to the
 
 You can add or remove any indices and index patterns as required. For background information on {es} indices, refer to {ref}/documents-indices.html[Data in: documents and indices].
 
-NOTE: If you leave the `logs-*` index selected, by default, all Elastic cloud logs are excluded from all queries in the {security-app}. This is to avoid adding data from cloud monitoring to the app.
+NOTE: If you leave the `logs-*` index pattern selected, by default, all Elastic cloud logs are excluded from all queries in the {security-app}. This is to avoid adding data from cloud monitoring to the app.
```

Comment:
This note was added in October 2020, and hasn't been updated since. Since then,`-*elastic-cloud-logs-*` was added as a default index pattern in `8.2`, so I'm not sure if this note is still correct. Explore team @elastic/security-threat-hunting-explore, as you've worked with the default indices in the past, do you happen to know this?

Reply:
@natasha-moore-elastic Yes you are right, `-*elastic-cloud-logs-*` index pattern is now part of the default Security Data View. This note can be confusing, can we remove it?

Reply:
Thanks for confirming, @semd! I've removed the note.

Reply:
Update: Discussed offline, the note is fine, we just need to replace `logs-*` by `-*elastic-cloud-logs-*`.